projects/ADuCM3029_demo_cn0503/src/adpd410x.c

```diff
@@ -76,11 +76,11 @@ int32_t adpd410x_reg_read(struct adpd410x_dev *dev, uint16_t address,
 }
 
 /**
- * @brief Read a single byte from device register.
+ * @brief Read a specified number of bytes from device register.

```

Comment:
Can we have these changes in a separate commit so we can propagate them later to the no-OS repository?